people/v_people.staff_roster.sql

```diff
@@ -95,6 +95,34 @@ WITH all_staff AS (
   GROUP BY associate_id
  )
 
+,survey_race_ethnicity AS (
+  SELECT employee_number
+        ,gabby.dbo.GROUP_CONCAT(answer) AS preferred_race_ethnicity
+  FROM gabby.surveys.staff_information_survey_detail
```

Comment:
can you review this view? for performance reasons, I'd really like to avoid attaching it to staff_roster

is there a reason it has it's own identifier CTE? it JOINs to survey_response_identifiers_static later on, so I'm unclear why